### DISCUSSION TITLE
Extra label button: 'Relevant for a different reason' or just use notes?

### DISCUSSION THREAD

**Discussion by RiannevanderFeen** [asreview/asreview#1943] [Ideas]
While reviewing the need arose to have a different filing system available in ASReview Lab. Besides being able to add notes (which allows for a separate labeling system in a sense), it would be good to include irrelevant articles as relevant but for a different reason, so that the model still includes certain aspects of the 'irrelevant, but relevant' articles.
 
As an example, we are now going over the relevant sample as an in-between step between phase 2 and 3 of the Safe procedure. With such a large sample size we assumed that we might have included too many articles as relevant and wanted to check this. 
We found that errors were made in terms of the inclusion criteria, but we also found that a subset of articles were included that could be relevant on the level of theory/frameworks/instruments. These articles will help write the background section, for example. These theories hold information that is relevant for the language model, but do not fit all of the inclusion criteria for the results section for the benefits of executing a systematic review. 

I guess including, but adding a note could work. But in that case the analytics in ASReview Lab are off (in case you want to use the images).

Hope this makes sense and I don’t yet have an answer.

**Comment by Rensvandeschoot** [comment]
Great idea!! I think the new tag functionality will suit your needs: https://github.com/asreview/asreview/pull/1527

@J535D165 any screenshots you would be willing to share how it may look like in v2, for inspiration and to get the discussion going?
